elasticdl/proto/elasticdl.proto

```diff
@@ -57,11 +57,14 @@ message Tensor {
 
     // Dtype of the tensor, e.g. "int64", "float32".
     TensorDtype dtype = 4;
+
+    // Tensor name
+    string name = 5;
```

Comment:
Just a convention, we usually put `name` to the first field.

Reply:
Done